.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/fsfe/reuse-tool
-    rev: v0.14.0
+    rev: v1.0.0
```

Comment:
I think we want to keep this on `v0.14.0` for now. When we update we'll patch it across all of the libraries at once. Until then they should stay the same as each other if possible.